react/week3/TodoDescription.js

```diff
@@ -0,0 +1,44 @@
+import React, { useState } from "react";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+
+function TodoDescription({ addTodo }) {
+  const [state, setState] = useState("");

```

Comment:
`state` is a very generic name. Please give it a more descriptive name
```suggestion
  const [description, setDescription] = useState("");

```